sympy/series/tests/test_limits.py

```diff
@@ -282,7 +282,7 @@ def test_set_signs():
     assert limit(abs(sin(x + 1)), x, 0) == sin(1)
 
     # https://github.com/sympy/sympy/issues/9449
-    assert limit((Abs(x + y) - Abs(x - y))/(2*x), x, 0) == sign(y)
+    assert limit((Abs(x + y) - Abs(x - y))/(2*x), x, 0) == 1/sign(y)
```

Comment:
This is not correct if y is not real. (The limit is real while `sign(y)` may be complex.) It should probably be `Re(y)/Abs(y)` (or maybe `Re(sign(y))`).